homeassistant/components/gpsd/sensor.py

```diff
@@ -98,6 +98,7 @@ class GpsdSensor(SensorEntity):
 
     _attr_has_entity_name = True
     _attr_name = None
+    _attr_translation_key = DOMAIN
```

Comment:
Please add device class enum and set the options

Reply:
Done, thanks.